### PR TITLE
Allow PHP 8.0

### PR DESCRIPTION
**Pull request by Yozhef**

### CODE REVIEW

**Comment by Yozhef** [comment]
@shadowhand can you please review?

**Comment by shadowhand** [comment]
@Yozhef please leave Github Actions out of this. I like the idea of switching, but it has nothing to do with PHP 8 support.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        branches: [ master ]
+
+jobs:
+    run:
+        runs-on: ubuntu-18.04
+        strategy:
+            fail-fast: false
+            matrix:
+                php:
+                    - '7.1'
```
Comment (shadowhand) [inline .github/workflows/ci.yaml:15]: PHP 7.1 can be dropped, as it is no longer supported upstream.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                minimum_versions: [false]
+                include:
+                    - description: 'Minimum version'
+                      php: '7.1'
```
Comment (shadowhand) [inline .github/workflows/ci.yaml:23]: Keep this, but target PHP 7.2.

File: .github/workflows/static-analysis.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+name: Code style
+
+on:
+    pull_request:
+    push:
+        branches: [ master ]
+
+jobs:
+    php-cs-fixer:
+        name: PHP-CS-Fixer
```
Comment (shadowhand) [inline .github/workflows/static-analysis.yaml:10]: This isn't correct. We are running `phpcs` to check code style, not fixing it. PHP-CS-Fixer is a different tool entirely.

File: composer.json [hunk 4]
```diff
@@ -19,6 +19,7 @@
     ],
     "minimum-stability": "stable",
     "require": {
+        "php": ">=7.1",
```
Comment (shadowhand) [inline composer.json:22]:
```suggestion
        "php": ">=7.2",
```

**Review by shadowhand — CHANGES_REQUESTED** [review]

**Comment by Yozhef** [comment]
@shadowhand

**Pull request closed without merging**